### PR TITLE
Refactor footer

### PR DESCRIPTION
**Pull request by Clindbergh**

### CODE REVIEW

File: assets/sass/components/_Container.scss [hunk 1]
```diff
@@ -4,12 +4,13 @@
  * This component can be used to center the main content.
  */
 
+$containerSidePadding: 1.5em;
```
Comment (ioboi) [inline assets/sass/components/_Container.scss]: This variable should be in _variables.scss

File: assets/sass/components/_Footer.scss [hunk 2]
```diff
@@ -1,71 +1,109 @@
 /*
  * Footer component
  */
-.Footer {
+footer {
```
Comment (ioboi) [inline assets/sass/components/_Footer.scss]: Please use a class for better maintanable CSS.

File: layouts/partials/footer/footer.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<footer>
+    <div>
```
Comment (ioboi) [inline layouts/partials/footer/footer.html]: Why wrap everything in a div?

File: layouts/partials/languages.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<ul class="LanguageSwitcher">
```
Comment (ioboi) [inline layouts/partials/languages.html]: The partial should be named LanguageSwitcher.

File: assets/sass/components/_Footer.scss [hunk 5]
```diff
@@ -1,71 +1,109 @@
...
+    @media screen and (max-width: $logoWidth + 2*$textWidth + 2*$containerSidePadding) {
+      justify-content: center;
+      flex-flow: column;
+      align-items: center;
+    }
 
-  div {
-    padding: 0.5em 0.8em;
-  }
+    a, svg {
```
Comment (ioboi) [inline assets/sass/components/_Footer.scss]: Please use explicit classes respecting BEM. I now this was not respected in other Pull-Requests but I suspect our CSS at the moment with all this nesting occurring is not maintainable anymore. We need to strive for a better structure and cleanup everything.

File: assets/sass/components/_Footer.scss (hunk 2 above)
Comment (Clindbergh) [reply assets/sass/components/_Footer.scss]: No problem, but why do you consider it more maintainable?

Comment (ioboi) [reply assets/sass/components/_Footer.scss]: [https://frontstuff.io/you-need-to-stop-targeting-tags-in-css](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

**Pull request merged**